examples/webxr_vr_ballshooter.html

```diff
@@ -90,7 +90,7 @@
 
 				//
 
-				renderer = new THREE.WebGLRenderer( { antialias: true } );
+				renderer = new THREE.WebGLRenderer( { antialias: true, multiviewStereo: true } );
```

Comment:
I don't think the developer should have to enable this.
Ideally the renderer would just use the extension when available.